packages/snaps-utils/src/manifest/validators/expected-files.ts

```diff
@@ -17,7 +17,10 @@ export const expectedFiles: ValidatorMeta = {
   structureCheck(files, context) {
     for (const expectedFile of EXPECTED_SNAP_FILES) {
       if (!files[expectedFile]) {
-        context.report(`Missing file "${SnapFileNameFromKey[expectedFile]}".`);
+        context.report(
+          'expected-files',
```

Comment:
Should this ID use `expectedFile`? Otherwise it is not unique

Reply:
Good point, will update.